packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -14,70 +14,103 @@ import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-i
 import UploadIcon from '@patternfly/react-icons/dist/esm/icons/upload-icon';
 import { Link } from '@reach/router';
 
-## Examples
+## Variants
 
-### Variations
+PatternFly supports several button styling variants to be used in different scenarios as needed. The following button variants can be assigned using the `variant` prop.
```

Comment:
In general is `prop` a common enough abbreviation or should we use `property` instead?